pygmt/tests/test_geopandas.py

```diff
@@ -141,6 +142,24 @@ def test_geopandas_plot3d_non_default_circle():
         "int64",
         pd.Int32Dtype(),
         pd.Int64Dtype(),
+        pytest.param(
+            "int32[pyarrow]",
+            marks=[
+                td.skip_if_no(package="pyarrow"),
+                pytest.mark.xfail(
+                    reason="geopandas doesn't support writing columns with pyarrow dtypes to OGR_GMT yet."
+                ),
+            ],
+        ),
+        pytest.param(
+            "int64[pyarrow]",
+            marks=[
+                td.skip_if_no(package="pyarrow"),
+                pytest.mark.xfail(
+                    reason="geopandas doesn't support writing columns with pyarrow dtypes to OGR_GMT yet."
+                ),
+            ],
+        ),
```

Comment:
So geopandas.GeoDataFrame objects with columns that have pyarrow dtypes can't be plotted in PyGMT yet, because we write them to a temporary OGR_GMT file, and `geopandas` doesn't support writing `int32[pyarrow]` or `int64[pyarrow]` columns. We'll either need to wait for `geopandas` to support this, or switch PyGMT away from using temporay OGR_GMT files.